### PR TITLE
Added translation scripts to integrate into LP translation system

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Added translation scripts
- Updated the configs to grab the right pages
- Ran then to create the `.po` files

## QA
- Just a code review required as its a mechanical branch

### CODE REVIEW

File: po-html/POTFILES.in [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Here you can specify the files to extract translations from, one per line.
+# For now, it will also be used to decide which files to merge translations
+# back to.
+# Lines starting with a hash will be interpreted as comments and be gfnored.
```
Comment (grahambancroft) [inline po-html/POTFILES.in]: should be ‘ignored'

**Comment by dplanella** [comment]
Following Ant's ping, a couple of comments:
- The `*.pyc` files should be removed from the commit, and probably added to `.gitignore`
- It might be worth checking if all the messages from the HTML files are indeed in the committed `.pot` file. At a quick glance, it seemed to be a small file compared to the amount of content we've got on snapcraft.io

File: po-html/POTFILES.in (hunk 1 above)
Comment (anthonydillon) [inline po-html/POTFILES.in]: Done, thanks.

**Comment by anthonydillon** [comment]
Thank you @dplanella I have removed and ignored all `.pyc` files. Also when through both pages verifying each piece of content is there which it is.

**Pull request merged**